### PR TITLE
Form validation benchmark added

### PR DESCRIPTION
**Pull request by deepakdinesh1123**

### CODE REVIEW

File: benchmarks/form_benchmarks/form_validate/benchmark.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        raise ValidationError
+
+class BookForm(forms.Form):
+    title = forms.CharField(max_length=100, validators=[form_validator])
+
+class FormValidate:
+
+    def setup(self):
+        bench_setup()
+        self.form = BookForm({'title':'hi'})
```
Comment (smithdc1) [inline benchmarks/form_benchmarks/form_validate/benchmark.py]: Nice -- should we add a benchmark for when a validation error is raised?

**Comment by smithdc1** [comment]
Could you rebase onto `main`? This diff is larger than I'm expecting and think a rebase will resolve it. 

Let me know if you need any help with this.

**Comment by deepakdinesh1123** [comment]
I reabsed the branch onto main and then committed the changes after that git gave me a message saying that there were changes in origin and I had to do a git pull before committing the changes, I did a git pull after which I pushed the commits, can you please tell me what I should have done?

**Comment by smithdc1** [comment]
Well, I'll have a go 👍 

I assume that you have your fork as `origin`, 

1. Add this repo as `upstream`. `git remote add upstream https://github.com/smithdc1/django-asv`

You can verify it is added correctly with `git remote -v`

2. Fetch `git fetch upstream`

3. Rebase `git rebase upstream/main`

4. At this point you'll need to resolve the merge conflicts. 

5. Continue the rebase `git rebase --continue`

6. Once this is complete your remote branch won't be inline with your local one. So when you push you'll get an error (maybe this is where you got to). You'll need to force to overwrite the remote. `git push --force`.

Hopefully that's it! 🤞

**Comment by smithdc1** [comment]
Thanks @deepakdinesh1123 👍 

Tests are working nicely too!

**Pull request merged**